lib/rspec/bash/command/call_configuration.rb

```diff
@@ -16,7 +16,7 @@ def set_exitcode(exitcode, args = [])
 
       def add_output(content, target, args = [])
         current_conf = create_or_get_conf(args)
-        current_conf[:outputs] << {
+        current_conf[:outputs][target] = {
           target: target,
```

Comment:
The reason the target is still left in the child object is so that the bash stub can still easily query for it. It's currently not able to pull the target out a key, so this works to help it pull it out of a value instead.